python/ray/data/tests/test_split.py

```diff
@@ -131,12 +131,12 @@ def test_equal_split_balanced_grid(ray_start_regular_shared):
     seed = int(time.time())
     print(f"Seeding RNG for test_equal_split_balanced_grid with: {seed}")
     random.seed(seed)
-    max_num_splits = 20
-    num_splits_samples = 5
+    max_num_splits = 15
+    num_splits_samples = 3
     max_num_blocks = 50
     max_num_rows_per_block = 100
-    num_blocks_samples = 5
-    block_sizes_samples = 5
+    num_blocks_samples = 3
+    block_sizes_samples = 3
```

Comment:
What's the expected # of combinations now?

Reply:
The multiplication of all the changed parameters.